app/upgrades/upgrades.go

```diff
@@ -336,3 +338,122 @@ func V010407UpgradeHandler(
 		return mm.RunMigrations(ctx, configurator, fromVM)
 	}
 }
+
+func V010600UpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	appKeepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	// TODO must add test and refactor current duplicated logic out of app/upgrades
+	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		migrations := map[string]string{
+			"quick1a7n7z45gs0dut2syvkszffgwmgps6scqen3e5l": "quick1h0sqndv2y4xty6uk0sv4vckgyc5aa7n5at7fll",
+			"quick1m0anwr4kcz0y9s65czusun2ahw35g3humv4j7f": "quick1n4g6037cjm0e0v2nvwj2ngau7pk758wtwk6lwq",
+		}
+
+		for fromBech32, toBech32 := range migrations {
+			from, err := addressutils.AccAddressFromBech32(fromBech32, "quick")
+			if err != nil {
+				return nil, err
+			}
+			to, err := addressutils.AccAddressFromBech32(toBech32, "quick")
+			if err != nil {
+				return nil, err
+			}
+			err = processMigratePeriodicVestingAccount(ctx, appKeepers, from, to)
+			if err != nil {
+				return nil, err
+			}
+		}
+
+		return mm.RunMigrations(ctx, configurator, fromVM)
+	}
+}
+
+// TODO: add logic collect rewards
+// processMigratePeriodicVestingAccount moves the unvested from current account ->  new account
+// - Unbonding all assets
+// - Claim rewards
+// - Post migrations
+func processMigratePeriodicVestingAccount(ctx sdk.Context, appKeepers *keepers.AppKeepers, from sdk.AccAddress, to sdk.AccAddress) error {
```

Comment:
Need add upgrades_test to make sure upgrade success

Reply:
Test added, I will add more test when I implement the claim reward before migration logic